pinot-plugins/pinot-file-system/pinot-gcs/src/test/java/org/apache/pinot/plugin/filesystem/GcsPinotFSTest.java

```diff
@@ -92,7 +92,7 @@ public void setup() {
     }
   }
 
-  @AfterClass
+  @AfterMethod
```

Comment:
could you elaborate more on why this change is needed? thx

Reply:
hey, gladly, when done with @AfterClass this particular test was failing, it's important to note that this test only runs if this is provided https://github.com/apache/pinot/blob/9f92392b902fb9f25ce0af63bae9d925a4b9525b/pinot-plugins/pinot-file-system/pinot-gcs/src/test/java/org/apache/pinot/plugin/filesystem/GcsPinotFSTest.java#L84 given that I made changes around that class I was running this test, and I found that because it was not getting cleaned up after every method this assertion was failing https://github.com/apache/pinot/blob/9f92392b902fb9f25ce0af63bae9d925a4b9525b/pinot-plugins/pinot-file-system/pinot-gcs/src/test/java/org/apache/pinot/plugin/filesystem/GcsPinotFSTest.java#L287 given that there are 2 files created by https://github.com/apache/pinot/blob/9f92392b902fb9f25ce0af63bae9d925a4b9525b/pinot-plugins/pinot-file-system/pinot-gcs/src/test/java/org/apache/pinot/plugin/filesystem/GcsPinotFSTest.java#L156 that are not cleaned up. 

So I decided to run the clean up after each test method is invoked.

Reply:
in this case I think we should split this into before/after class and before after method. it is super weird to have beforeclass then after method.

Reply:
hmm. github doesn't allow me to comment outside of range. can I directly push to your branch?

Reply:
yea feel free to do so :)

Reply:
cant push to your branch b/c write access issue, here is the fix: https://github.com/apache/pinot/commit/caa1caf9e3c5cd12fde7853ec216d190d7f04deb

Reply:
made the change, thank you @walterddr how can i trigger a new test build here?

Reply:
just did

Reply:
seems like everything came out okay! :)

Reply:
yeah, the test is skipped b/c there's no credential on GHA. let me verify locally

Reply:
i did run locally and it works

Reply:
do we need anyone else review?